user_modules/shared_testcases/testCasesForExternalUCS.lua

```diff
@@ -0,0 +1,179 @@
+    utils.pts.policy_table.consumer_friendly_messages.messages = nil
+    utils.pts.policy_table.device_data = nil
+    utils.pts.policy_table.module_meta = nil
+    utils.pts.policy_table.usage_and_error_counts = nil
+    utils.pts.policy_table.app_policies[appId] = {
+      keep_context = false,
+      steal_focus = false,
+      priority = "NONE",
+      default_hmi = "NONE"
+    }
+    utils.pts.policy_table.app_policies[appId]["groups"] = { "Base-4", "Base-6" }
+    utils.pts.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
+    utils.pts.policy_table.module_config.preloaded_pt = nil
+  end
+
+	local function ptu(test, id, status)
+    local policy_file_name = "PolicyTableUpdate"
+    local policy_file_path = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath")
+    local ptu_file_name = os.tmpname()
+    local requestId = test.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+    EXPECT_HMIRESPONSE(requestId)
+    :Do(
+      function()
+        test.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+          { requestType = "PROPRIETARY", fileName = policy_file_name })
+        utils.createJsonFileFromTable(utils.pts, ptu_file_name)
+        EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UPDATING" }, { status = status }):Times(2)
+        test["mobileSession"..id]:ExpectNotification("OnSystemRequest", { requestType = "PROPRIETARY" })
+        :Do(
+          function()
+	          local corIdSystemRequest = test["mobileSession"..id]:SendRPC("SystemRequest",
+	            { requestType = "PROPRIETARY", fileName = policy_file_name }, ptu_file_name)
+	          EXPECT_HMICALL("BasicCommunication.SystemRequest")
+	          :Do(
+	            function(_, d)
+	              test.hmiConnection:SendResponse(d.id, "BasicCommunication.SystemRequest", "SUCCESS", { })
+	              test.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+	                { policyfile = policy_file_path .. "/" .. policy_file_name })
+	            end)
+	            test["mobileSession"..id]:ExpectResponse(corIdSystemRequest, { success = true, resultCode = "SUCCESS"})
+          end)
+      end)
+    os.remove(ptu_file_name)
+  end
+
+	function utils.startSession(test, id)
+    test["mobileSession"..id] = mobile_session.MobileSession(test, test.mobileConnection)
+    test["mobileSession"..id]:StartService(7)
+  end
+
+	function utils.registerApp(test, id)
+		local RAIParams = config["application"..id].registerAppInterfaceParams
+    local corId = test["mobileSession"..id]:SendRPC("RegisterAppInterface", RAIParams)
+    EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered",
+      { application = { appName = RAIParams.appName } })
+    :Do(
+      function(_, d)
+        utils.HMIAppIds[RAIParams.appID] = d.params.application.appID
+      end)
+    test["mobileSession"..id]:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+    :Do(
+      function()
```

Comment:
@dboltovskyi, in scripts usually function is listed in the same row as :Do(
:Do(function()